src/definitions/modules/slider.less

```diff
@@ -254,109 +254,78 @@
      Colors
 ---------------*/
 
-/* Red */
-.ui.red.slider .inner .track-fill {
-  background-color: @red;
-}
-.ui.red.inverted.slider .inner .track-fill {
-  background-color: @lightRed;
-}
+.addColor(@color, @lightColor, @hoverColor, @lightHoverColor) {
```

Comment:
The last 2 parameters for addColor (hover+lighthover) are not used anymore since you removed the code for "basic"
Thus all later calls to addColor can also be reduced by those parameters

Reply:
forget about this,  i was blind...